src/base/utils/password.cpp

```diff
@@ -99,12 +99,12 @@ bool Utils::Password::PBKDF2::verify(const QByteArray &secret, const QString &pa
 
 bool Utils::Password::PBKDF2::verify(const QByteArray &secret, const QByteArray &password)
 {
-    const QVector<QByteArray> list = ByteArray::splitToViews(secret, ":", Qt::SkipEmptyParts);
+    const QList<QByteArrayView> list = ByteArray::splitToViews(secret, ":", Qt::SkipEmptyParts);
     if (list.size() != 2)
         return false;
 
-    const QByteArray salt = QByteArray::fromBase64(list[0]);
-    const QByteArray key = QByteArray::fromBase64(list[1]);
+    const QByteArray salt = QByteArray::fromBase64(list[0].toByteArray());
+    const QByteArray key = QByteArray::fromBase64(list[1].toByteArray());
```

Comment:
I would create an utility function to efficiently convert an QByteArrayView to an QByteArray, otherwise it will result in deep copy (which should be regarded as an performance regression of this PR).

Reply:
> I would create an utility function to efficiently convert an QByteArrayView to an QByteArray, otherwise it will result in deep copy

How can you create "regular" QByteArray from view without real copying the data?

Reply:
Of course we can still use "weak" `QByteArray` (created using `QByteArray::fromRawData()`) to pass into the functions that accept `const QByteArray &` and haven't still overloads that accept `QByteArrayView`. If that's what you mean, then I wouldn't use the term "convert" in the code, but something like `asQByteArray(QByteArrayView view)` to try to emphasize the fact that we are not talking about creating a "regular" `QByteArray` that owns the data.